### PR TITLE
Add max 50 character limit to text input fields

### PR DESCRIPTION
**Pull request by CharlBest**
Profile name, company and website inputs should have some upper limit

From the Svelte docs:
image

Example:
image

### CODE REVIEW

**Comment by owenversteeg** [comment]
Probably a good idea to check this in the backend too, not that it's that "dangerous" but just a good philosophy to have IMO.

**Comment by scosman** [comment]
Headed out on a vacation, but thanks for patches. Not ignoring these. Will look at this and the other one when I'm back!

**Comment by scosman** [comment]
As @owenversteeg mentioned, would want to add these checks to server side too in `api/+page.server.ts:updateProfile`. There are already examples in that file of checking length, should be easy addition.

File: src/routes/(admin)/account/api/+page.server.ts [hunk 1]
```diff
@@ -203,20 +203,30 @@ export const actions = {
...
+    } else if (companyName.length > fieldMaxTextLength) {
+      validationError = `Company name must be less than ${fieldMaxTextLength} charaters`
+      errorFields.push("companyName")
     }
     if (!website) {
       validationError =
         "Company website is required. An app store URL is a good alternative if you don't have a website."
       errorFields.push("website")
+    } else if (website.length > fieldMaxTextLength) {
+      validationError = `Company website must be less than ${fieldMaxTextLength} charaters`
```
Comment (scosman) [inline src/routes/(admin)/account/api/+page.server.ts:228]: typo: characters (3x)

**Review by scosman — COMMENTED** [review]
one nit

**Comment by scosman** [comment]
Merging into another branch so CI runs and it checks linting/formatting/etc. If that all works, lgtm. 

I've fixed it so CI should run on future external PRs here: https://github.com/CriticalMoments/CMSaasStarter/commit/8c02fcb9c04d029b5ee2bcca6addb051dba89876

**Pull request merged**